lib/travis/build/appliances/update_mongo_arch.rb

```diff
@@ -0,0 +1,22 @@
+require 'travis/build/appliances/base'
+
+module Travis
+  module Build
+    module Appliances
+      class UpdateMongoArch < Base
+        def apply
+          command = <<-EOF
+            if command -v lsb_release &>/dev/null && [[ $(lsb_release -sc) != precise ]]; then
```

Comment:
Why is Precise excluded from this list? (I'm not saying that we should keep supporting Precise, but that this extra condition seems out of place.)